### DISCUSSION TITLE
Does NanoMQ plan to support QUIC?

### DISCUSSION THREAD

**Discussion by skawu** [nanomq/nanomq#746] [Q&A] [answered]
看到最新发布的0.9.0中，NanoSDK支持了QUIC，作为Broker，NanoMQ是否有计划支持QUIC

**Comment by JaylinYu** [comment, marked as answer]
nanomq作为主要针对边缘的broker+message bus，quic并非最高优先级。不过会马上支持quic 桥接，可以让没有quic sdk的客户端通过桥接使用quic

**Reply by skawu** [reply]
感谢回复！

**Comment by yangbinsanqianchi** [comment]
如果nanomq也能支持quic就好了，也可以做云服务用
